pkgs/development/python-modules/pyft232/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pyserial
+, pyusb
+}:
+
+buildPythonPackage rec {
+  pname = "pyft232";
+  version = "0.9";
+
+  src = fetchFromGitHub {
+    owner = "lsgunth";
+    repo = "${pname}";
```

Comment:
usage of pname is considered anti-pattern https://github.com/NixOS/nixpkgs/issues/277994